website/public/js/controllers/tasksCtrl.js

```diff
@@ -129,6 +132,24 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
 
     /*
      ------------------------
+     Dailies
+     ------------------------
+     */
+    $scope.updateTaskStartDate = function(task) {
+      // TODO: Weirdness here...startDate should be a Date and _dateString
+      // should be a string, but the datePicker input fields sets _dateString
+      // to a Date even though input type="text"...
+      task.startDate = task._dateString;
+    };
+
+    $scope.open = function($event) {
```

Comment:
1. and 2. both addressed https://github.com/HabitRPG/habitrpg/commit/2028037d2ce5dcf17bdcbc77da499e7dffcc0bf6